brianlong/www.validators.app
# 180894747 - display active stake instead of software version

#### What's this PR do?
PR adds the active stake total onto the stake pool index page.

#### How should this be manually tested?
Changes concerns the stake pool page, especially the validator details part.

- check the validator's details in the stake pool page if does includes the active stake total number and excludes the software version
- to populate the stake accounts you may run the `rails r script/gather_stake_accounts.rb`

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/180894747)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
